mmcv/ops/deform_conv.py

```diff
@@ -210,7 +210,11 @@ class DeformConv2d(nn.Module):
         deform_groups (int): Number of deformable group partitions.
         bias (bool): If True, adds a learnable bias to the output.
             Default: False.
-
+        im2col_step(int): Number of samples processed by the im2col_cuda_kernel
+        per call. It will work when batch_size > im2col_step, but batch_size
+        must be divisible by im2col_step. Solved this issue
```

Comment:
Indicate the next release version when this argument is added like `Added in v1.3.1x`

Reply:
Better to add a note section to discuss the im2col_step and attach the issue link there.

Reply:
Reference
https://github.com/open-mmlab/mmcv/blob/e22740b1d6953d75a0acecce4455d23800b1f018/mmcv/runner/hooks/evaluation.py#L64

`New in version 1.3.17.`